tools/testkit/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#![allow(dead_code)]
```

Comment:
Please don't do this. If you think you need something like this, you need a comment in the code explaining _why_.